src/main/java/com/google/sps/utils/GameUtils.java

```diff
@@ -123,4 +124,59 @@ public static boolean addUserToGame(String userId, Entity gameEntity, DatastoreS
     
     return true;
   }
-}
\ No newline at end of file
+ 
+  public static String joinGame(Entity userEntity, String gameName, DatastoreService datastore, String gameId) {
+ 
+    if(gameName == ""){
+        return "noGameName";
+    }
```

Comment:
There's no need for this check. Currently it's already impossible to create a game with an empty name. But if you ever change your mind and allow empty games, you'll need to delete this check. So this check only hurts you.

Reply:
I suggested this check because you'll never want to create a game with an empty name (from a usability standpoint) so it's not bad to make sure it's not happening. 

Why do you think it's impossible to create a game with an empty name?

Reply:
I assumed `joinGame` was a function for joining an existing game. Does it also create a game? If so it probably shouldn't

Reply:
Sounds good, I removed this check.